ui/src/literature/components/ImprintInfo.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { List } from 'immutable';
+
+import InlineList, { SEPARATOR_AND } from '../../common/components/InlineList';
+
+class ImprintInfo extends Component {
+  static renderImprints(imprints) {
+    const placePart = imprints.get('place')
```

Comment:
Maybe you can destructure this object before creating `placePart`, `datePart` etc.? It would be easier to read I think.